CHANGELOG.md

```diff
@@ -82,7 +84,7 @@ Performance:
 * Reduce memory footprint by tracking `VALUE`s created during C extension init separately (@aardvark179).
 * Rewrote `ArrayEachIteratorNode` to optimize performance for a constant-sized array and reduce specializations to 1 general case (#2587, @MattAlp)
 * Reduce conversion of `VALUE`s to native handle during common operations in C extensions (@aardvark179).
-* Improved performance of regex boolean matches (e.g., `Regexp#match?`) by avoiding match data allocation in TRegex (#2558, @nirvdrum).
+* Improved performance of regex boolean matches (e.g., `Regexp#match?`) by avoiding match data allocation in TRegex (#2588, @nirvdrum).
```

Comment:
I think it's preferred to only link to Github Issues:
https://github.com/oracle/truffleruby/commit/cd8b8824633cd9b5ae5f0c808c764c8f77f6f966

So, if there isn't any Github issue this could just be: `(@nirvdrum)`

Reply:
For old changelog entries it's fine to have a PR number (here it was incorrect PR number, so good fix), it doesn't hurt.